onnxruntime/core/framework/debug_node_inputs_outputs_utils.cc

```diff
@@ -215,7 +222,7 @@ const NodeDumpOptions& NodeDumpOptionsFromEnvironmentVariables() {
     opts.output_dir = Path::Parse(ToPathString(Env::Default().GetEnvironmentVar(env_vars::kOutputDir)));
 
     // check for confirmation for dumping data to files for all nodes
-    if (opts.dump_flags != NodeDumpOptions::DumpFlags::ShapeOnly &&
+    if (opts.dump_flags != NodeDumpOptions::DumpFlags::Shape &&
```

Comment:
this probably needs to be updated to check that either InputData or OutputData is dumped